### PR TITLE
Chore/move signer logs to info

### LINKED ISSUE #5001: Move Most Relevant Signer Logs to be displayed on Info instead of Debug
**Is your feature request related to a problem? Please describe.**
If the signer is run with logs=info, a lot of relevant logs are not shown.

**Describe the solution you'd like**
 This will move only what is most relevant from being displayed only on `debug`, to also be displayed on `info`.

### PR DESCRIPTION
**Pull request by ASuciuX**
### Description
Moves from `debug` to `info` some of the most relevant logs in order to be displayed on signers with the default logs level. If switched to `debug` they would have to many logs and have to know how to read through them. 

### Applicable issues

- fixes #5001 

### Checklist

- [x] receive block proposal - is for stacks blocks
- [x] <strike>validate block proposal - is for stacks blocks </strike> 
 [already implemented](https://github.com/stacks-network/stacks-core/blob/861bff076b051e0338b37dacbe0a17cd1b90b3b3/stacks-signer/src/v1/signer.rs#L657)
- [x] <strike>send signature </strike> - [already implemented](https://github.com/stacks-network/stacks-core/blob/861bff076b051e0338b37dacbe0a17cd1b90b3b3/stacks-signer/src/v1/signer.rs#L1331)
- [x] receive burn block

### CODE REVIEW

**Comment by ASuciuX** [comment]
@obycode in the first commit I also added the block proposal and block validation, but those are related to stacks blocks and it will display pretty frequent if we include them.

**Review by wileyj — DISMISSED** [review]

**Comment by obycode** [comment]
I'd prefer to keep those stacks block logs as `info`.

File: stacks-signer/src/v0/signer.rs [hunk 1]
```diff
@@ -251,7 +251,7 @@ impl Signer {
         block_proposal: &BlockProposal,
         miner_pubkey: &Secp256k1PublicKey,
     ) {
-        debug!("{self}: Received a block proposal: {block_proposal:?}");
+        info!("{self}: Received a block proposal: {block_proposal:?}");
```
Comment (obycode) [inline stacks-signer/src/v0/signer.rs:254]: I guess this one can remain a debug since the next log gives the same info and more.

Comment (obycode) [reply stacks-signer/src/v0/signer.rs:254]: They are both still info, right?

**Review by wileyj — APPROVED** [review]

File: stacks-signer/src/v0/signer.rs (hunk 1 above)
Comment (obycode) [reply stacks-signer/src/v0/signer.rs:254]: Let's make this one `debug!`.

Comment (ASuciuX) [reply stacks-signer/src/v0/signer.rs]: My bad, I thought that it had a suggestion on it and I wanted to confirm the change in the code. But I only clicked resolve comment. I've pushed it now.

**Review by jferrant — APPROVED** [review]

**Review by wileyj — APPROVED** [review]
lgtm

**Review by obycode — APPROVED** [review]
thanks!

**Pull request merged**